components/brave_perf_predictor/browser/p3a_bandwidth_savings_permanent_state.h

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_P3A_BANDWIDTH_SAVINGS_PERMANENT_STATE_H_
+#define BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_P3A_BANDWIDTH_SAVINGS_PERMANENT_STATE_H_
+
+#include <list>
+
+#include "base/timer/timer.h"
+
+class PrefService;
+
+namespace brave_perf_predictor {
+
+class P3ABandwidthSavingsPermanentState {
```

Comment:
mb a short comment saying what's happening

Reply:
followed the convention from `p3a_core_metrics.h`

Reply:
added some